Topic-3-JSDOM/1.B.js

```diff
@@ -0,0 +1,39 @@
+let daysWeek = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
```

Comment:
This one could be a constant, no changes will be done in that one. `const daysOfWeek = ...`